### PR TITLE
Sendgrid email setup

### PR DESCRIPTION
**Pull request by gbudjeakp**
Completed Todos:
- [x] setup sendgrid account
- [x] create a route that sends emails
- [x] share sendgrid API credentials with team on Slack

### CODE REVIEW

File: server/routes/email.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  const {
+    passenger,
+    from,
+    to,
+    departureTime,
+    arrivalTime,
+    hotelName,
+  } = flightData;
+
+  let msg = {
```
Comment (johnsciutto) [inline server/routes/email.js]: This could be a `const`.

File: server/routes/email.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    //  Chnage this to your own email to get mail sent to your inbox
+    to: "gbudjeakp@gmail.com",
+
+    //Do not change verified (The from email) Sender as SendGrid will only send from this email.
+    from: "sebdwebdev@gmail.com",
```
Comment (johnsciutto) [inline server/routes/email.js]: You probably don't want this on GitHub...

Comment (gbudjeakp) [reply server/routes/email.js]: I'll get rid of that ASAP.

File: server/routes/email.js (hunk 1 above)
Comment (gbudjeakp) [reply server/routes/email.js]: Making the change now

**Review by sina-jamshidi — APPROVED** [review]
looks good! But let's refactor this to be a generalized email sender service. Then we can call it from whatever route we want to whatever email with whatever message we want!

**Pull request merged**